README.md

```diff
@@ -375,10 +375,10 @@ The frontend now has a new client to retrieve Fibonacci numbers. Against the gen
 
 ## Task 4: Developing a new service endpoint using cloud contracs
 ### Introduction
-Now that the client is working the producer needs to implement the service. We've build the numbers-service code on our consumer side with `mvn clean install -DskipTests`. The reason for this, is that we wanted the stubs to be generated by the build process. Cloud contracts also generated unit-tests based on the contract on the producer side. Normally the consumer would submit a new contract through a pull request. Let's assume you are now the implementor of the producer, and you've just checked out the pull request.
+Now that the client is working the producer needs to implement the service. We've build the numbers-service code on our consumer side with `mvn clean install -DskipTests`. The reason for this is that we wanted the stubs to be generated by the build process. Cloud contracts also generated unit-tests based on the contract on the producer side. Normally the consumer would submit a new contract through a pull request. Let's assume you are now the implementor of the producer, and you've just checked out the pull request.
```

Comment:
@JohannisK If you use newlines more frequently in markdown it's easier to read diffs. As long as you use just one newline it won't show in the rendered text.